packages/rest-generator/src/openapi-types.ts

```diff
@@ -0,0 +1,65 @@
+import { OpenAPIV3 } from 'openapi-types';
+
+/**
+ * @experimental This API is experimental and might change in newer versions. Use with caution.
+ * Representation of a parsed OpenApi Service.
+ */
+export interface OpenApiDocument {
+  apiName: string;
+  serviceDirName: string;
+  operations: OpenApiOperation[];
+}
+
+/**
+ * @experimental This API is experimental and might change in newer versions. Use with caution.
+ * Representation of an operation.
+ */
+export interface OpenApiOperation extends OpenAPIV3.OperationObject {
+  operationName: string;
+  method: string;
+  pattern: string;
```

Comment:
I do not see where the `pattern` is used.

Reply:
In the `parseOperation` the pattern is parse explicitly.

Reply:
You are right, this is not used anywhere (yet). I think we might want to keep it anyways, because we will probably use it in the generated documentation later on (TSDoc for the generated functions).